tests/Unit/Evolution/Particles/MonteCarlo/Test_EvolvePackets.cpp

```diff
@@ -170,3 +172,208 @@ SPECTRE_TEST_CASE("Unit.Evolution.Particles.MonteCarloEvolution",
   CHECK(fabs(get(coupling_rho_ye)[ext_idx_0 + 1] + proton_mass * 5.0e-61)
     < 1.e-72);
 }
+
+tnsr::I<DataVector, 3, Frame::ElementLogical> spatial_coords_logical(
+    const Mesh<3>& mesh) {
+  const DataVector used_for_size(mesh.number_of_grid_points());
+  auto x = make_with_value<tnsr::I<DataVector, 3, Frame::ElementLogical>>(
+      used_for_size, 0.0);
+  const Index<3>& extents = mesh.extents();
+  Index<3> cur_extents{0, 0, 0};
+  for (cur_extents[0] = 0; cur_extents[0] < extents[0]; cur_extents[0]++) {
+    for (cur_extents[1] = 0; cur_extents[1] < extents[1]; cur_extents[1]++) {
+      for (cur_extents[2] = 0; cur_extents[2] < extents[2]; cur_extents[2]++) {
+        const size_t storage_index = mesh.storage_index(cur_extents);
+        x.get(0)[storage_index] =
+            -1.0 + 2.0 * static_cast<double>(cur_extents[0] + 0.5) /
+                       static_cast<double>(extents[0]);
+        x.get(1)[storage_index] =
+            -1.0 + 2.0 * static_cast<double>(cur_extents[1] + 0.5) /
+                       static_cast<double>(extents[1]);
+        x.get(2)[storage_index] =
+            -1.0 + 2.0 * static_cast<double>(cur_extents[2] + 0.5) /
+                       static_cast<double>(extents[2]);
+      }
+    }
+  }
+  return x;
+}
+tnsr::I<DataVector, 3, Frame::Inertial> spatial_coords_inertial(
+    tnsr::I<DataVector, 3, Frame::ElementLogical> logical_coords) {
+  auto x = make_with_value<tnsr::I<DataVector, 3, Frame::Inertial>>(
+      logical_coords.get(0), 0.0);
+  x.get(0) = logical_coords.get(0) + 6.0;
+  x.get(1) = logical_coords.get(1);
+  x.get(2) = logical_coords.get(2);
+  return x;
+}
+
+void test_evolve_kerr(const size_t mesh_size_1d) {
```

Comment:
Add here a comment of the type
// Evolve a single Monte-Carlo packet along a geodesic of Kerr, and check that final position matches expectations.